gupb_queue/views.py

```diff
@@ -2,20 +2,21 @@
 
 from gupb_queue.forms import QueueForm
 from gupb_queue.models import Queue
+from platforms.models import Platform
 from tournament.models import Tournament
 
 from . import queue_utils
+from platforms import platform_utils
 
 
 def show_queue_terms_view(request, queue_id):
```

Comment:
nie naciskam ale proponuje powoli przechodzić na class based views (między innymi po to żeby w każdej funkcji widoku nie było tego przeklętego request.method == POST oraz żeby kod był bardziej reużywalny)